pkg/pgmodel/ingestor/trace/cache.go

```diff
@@ -26,3 +28,76 @@ func newInstrumentationLibraryCache() *clockcache.Cache {
 func newTagCache() *clockcache.Cache {
 	return clockcache.WithMax(tagCacheSize)
 }
+
+func registerMetrics(cacheName string, c *clockcache.Cache) {
+	enabled := prometheus.NewGauge(
```

Comment:
Hmm... Isn't caching always on? Whats the use-case with having no cache? I question the value of this metric.